pkg/utils/helper.go

```diff
@@ -251,20 +251,15 @@ func GetCreationTimeStamp() (string, error) {
 }
 
 // WriteToCRD writes diagnostic data to CRD
-func WriteToCRD(fileName string, key string) error {
+func WriteToCRD(fileContent string, key string) error {
```

Comment:
Can we rename the parameter to crdContent or similar? This doesn't represent a file anymore